src/styles/card.scss

```
@@ -0,0 +1,119 @@
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  background-color: #FFF;
```

don't forget to create variables for all colors
```suggestion
  background-color: $someColor;
```